test/jdk/java/lang/module/ModuleDescriptorTest.java

```diff
@@ -1361,7 +1361,6 @@ public void testReadsWithPackageFinder() throws Exception {
      * Test ModuleDescriptor with a packager finder that doesn't return the
      * complete set of packages.
      */
-    @Test(expectedExceptions = InvalidModuleDescriptorException.class)
```

Comment:
At first I thought you accidentally removed the entire `@Test` annotation instead of just the `expectedExceptions` property. But then this class is annotated with a `@Test` and this method name starts with the `test...`, just like some other existing methods in this class which too don't have a `@Test` annotation. I checked a local run of this test and those other non-annotated methods are currently executed as test methods, which is a good thing. So this change to completely remove the `@Test` is fine.